scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -35,6 +35,11 @@
     become: true
     become_user: ardana
 
+  - name: Copy authorized keys from root to ardana user on deployer
+    shell: |
+      cat ~root/.ssh/authorized_keys ~ardana/.ssh/authorized_keys | sort -u  > ~ardana/.ssh/authorized_keys
+    become: true
```

Comment:
Won't this result in the file being owned by `root` rather than `ardana`?  And if it's `0640` (which it should be) then the `ardana` user won't be able to read it.

Reply:
a '''>''' doesn't change ownership, and something at least in my testing seem to have created the file already before hand. I've updated the previous tasklet with an explicit touch.